client.go

```diff
@@ -39,11 +39,17 @@ func (c *Client) Ports() ([]*Port, error) {
 	return c.c.Ports()
 }
 
+// DpipeTables retrieves all devlink DPIPE tables attached to specified device on this system.
+func (c *Client) DpipeTables(dev *Device) ([]*DpipeTable, error) {
```

Comment:
Should be "DPIPE" throughout per typical Go style, according to the doc I just found at https://github.com/Mellanox/mlxsw/wiki/Pipeline-Debugging-(DPIPE).

Reply:
Fixed.